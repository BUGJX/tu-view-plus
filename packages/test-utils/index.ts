export { default as defineGetter } from './define-getter';
