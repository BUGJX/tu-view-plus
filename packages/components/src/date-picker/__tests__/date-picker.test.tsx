import { mount } from '@vue/test-utils'
import { describe, expect, test } from 'vitest'
import DatePicker from '../src/pickers/date-picker.vue'

describe('DatePicker.vue', () => {
  test('render test', () => {
    const wrapper = mount(() => <DatePicker></DatePicker>)
    expect(wrapper.html()).toMatchSnapshot();
  })
})
