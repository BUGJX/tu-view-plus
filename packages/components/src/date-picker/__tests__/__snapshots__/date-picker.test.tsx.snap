// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DatePicker.vue > render test 1`] = `
"<div class=\\"tu-picker\\" visible=\\"false\\">
  <!--v-if-->
  <div class=\\"tu-picker__input-wrap\\"><input placeholder=\\"Please select date\\" class=\\"tu-picker__input\\"></div>
  <div class=\\"tu-picker__suffix\\">
    <!--v-if--><i class=\\"tu-icon tu-picker__icon\\">
      <!--v-if--><i class=\\"tu-icon\\">
        <!--v-if--><svg xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 1024 1024\\">
          <path fill=\\"currentColor\\" d=\\"M128 384v512h768V192H768v32a32 32 0 1 1-64 0v-32H320v32a32 32 0 0 1-64 0v-32H128v128h768v64H128zm192-256h384V96a32 32 0 1 1 64 0v32h160a32 32 0 0 1 32 32v768a32 32 0 0 1-32 32H96a32 32 0 0 1-32-32V160a32 32 0 0 1 32-32h160V96a32 32 0 0 1 64 0v32zm-32 384h64a32 32 0 0 1 0 64h-64a32 32 0 0 1 0-64zm0 192h64a32 32 0 1 1 0 64h-64a32 32 0 1 1 0-64zm192-192h64a32 32 0 0 1 0 64h-64a32 32 0 0 1 0-64zm0 192h64a32 32 0 1 1 0 64h-64a32 32 0 1 1 0-64zm192-192h64a32 32 0 1 1 0 64h-64a32 32 0 1 1 0-64zm0 192h64a32 32 0 1 1 0 64h-64a32 32 0 1 1 0-64z\\"></path>
        </svg>
        <!--v-if-->
      </i>
      <!--v-if-->
    </i>
  </div>
</div>"
`;
